### DISCUSSION TITLE
Responsiveness of stars tab.

### DISCUSSION THREAD

**Discussion by george-bobby** [community/community#8673] [Other Feature Feedback, Questions, & Ideas] [answered]
It is looking fine on desktop. But I think the whole stars tab isn't responsive. 
ie `https://github.com/YOUR_USERNAME?tab=stars`
![Screenshot_2021-12-10-22-30-13-418_com brave browser](https://user-images.githubusercontent.com/74462795/145612609-1b7ef90e-1c99-486a-b1e6-1d44b6032516.png)
And I think the sort by drop-down menu is the culprit.

But it is responsive if we take `https://github.com/stars/YOUR_USERNAME/lists/YOUR_LIST`

PS: Lists are awesome for people like me who stars a lot of repos.

Ref : #8660

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up! We've opened an issue to track this, we'll try to get it fixed soon 😊

**Comment by george-bobby** [comment]
The issue is now fixed.
